cea/tests/test_district_cooling.py

```diff
@@ -0,0 +1,200 @@
+        cls.slave_variables.WasteServersHeatRecovery = \
+            json.loads(cls.test_config.get("input_heating_technologies", "WasteServersHeatRecovery"))
+
+        # COOLING TECHNOLOGIES
+        # NG-fired trigen
+        cls.slave_variables.NG_Trigen_on = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "NG_Trigen_on"))
+        cls.slave_variables.NG_Trigen_ACH_size_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "NG_Trigen_ACH_size_W"))
+        cls.slave_variables.NG_Trigen_CCGT_size_thermal_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "NG_Trigen_CCGT_size_thermal_W"))
+        cls.slave_variables.NG_Trigen_CCGT_size_electrical_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "NG_Trigen_CCGT_size_electrical_W"))
+
+        # Water-source vapour compression chillers
+        cls.slave_variables.WS_BaseVCC_on = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "WS_BaseVCC_on"))
+        cls.slave_variables.WS_BaseVCC_size_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "WS_BaseVCC_size_W"))
+
+        cls.slave_variables.WS_PeakVCC_on = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "WS_PeakVCC_on"))
+        cls.slave_variables.WS_PeakVCC_size_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "WS_PeakVCC_size_W"))
+
+        # Air-source vapour compression chiller
+        cls.slave_variables.AS_BaseVCC_on = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "AS_BaseVCC_on"))
+        cls.slave_variables.AS_BaseVCC_size_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "AS_BaseVCC_size_W"))
+
+        cls.slave_variables.AS_PeakVCC_on = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "AS_PeakVCC_on"))
+        cls.slave_variables.AS_PeakVCC_size_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "AS_PeakVCC_size_W"))
+
+        cls.slave_variables.AS_BackupVCC_on = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "AS_BackupVCC_on"))
+        cls.slave_variables.AS_BackupVCC_size_W = \
+            json.loads(cls.test_config.get("input_cooling_technologies", "AS_BackupVCC_size_W"))
+
+        # Storage Cooling
+        cls.slave_variables.Storage_cooling_on = \
+            json.loads(cls.test_config.get("input_storage", "Storage_cooling_on"))
+        cls.slave_variables.Storage_cooling_size_W = \
+            json.loads(cls.test_config.get("input_storage", "Storage_cooling_size_W"))
+
+    def test_cooling_main(self):
+        # config-project directory to the reference case
+        project_reference_case = os.path.join(tempfile.gettempdir(), 'reference-case-open')
+        self.config.__setattr__('project', project_reference_case)
```

Comment:
You should be able to just call `self.config.project`

Reply:
Thanks. I integrated your suggestion.